backend/app/services/dsh_plugin_storage.py

```
@@ -0,0 +1,321 @@
+        limit: int,
+    ) -> List[Dict[str, Any]]:
+        self.validate_identity(package_name, unit)
+        self._validate_name(table, "table")
+        rows = (
+            db.query(Kind, User)
+            .join(User, User.id == Kind.user_id)
+            .filter(
+                Kind.kind == DSH_PLUGIN_DATA_KIND,
+                Kind.namespace == package_name,
+                Kind.name == unit,
+                Kind.is_active == True,
```

Ruff (0.16.2)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)